sentry_dsn_data.go

```diff
@@ -0,0 +1,163 @@
+package main
+
+import (
+	"context"
+	"errors"
+	"sync"
+
+	"github.com/getsentry/sentry-go"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+var DSN = "dsn"
```

Comment:
Let's rename both the variable (make it more verbose/less ambigusou, something like `DSNAnnotation`) and the annotation itself (e.g. `k8s.sentry.io/dsn`)